### DISCUSSION TITLE
How to use withOpacity

### DISCUSSION THREAD

**Discussion by vmatekole** [satnaing/astro-paper#265] [FAQ]
How to use [withOpacity](https://github.com/satnaing/astro-paper/blob/8e022c7b5ab6751432fee34adc23e30f2ec8b74c/tailwind.config.cjs#L1) function with CSS variables? It isn't clear how to use this function and pass the opacity value?

**Comment by satnaing** [comment]
In TailwindCSS v2, we were able to use something like`bg-opacity-25` instead of changing the entire `opacity` value.

In order to integrate this kind of feature using CSS variable, we need to use `withOpacity` helper function to achieve the same effect.
If you're interested, you can deep dive into the reason behind this function. Here's [the original video](https://www.youtube.com/watch?v=MAtaT8BZEAo&t=524s) from Tailwind YouTube.

In AstroPaper, this function is not even exported. Which means you don't have to use it elsewhere except `tailwind.config.cjs`.
